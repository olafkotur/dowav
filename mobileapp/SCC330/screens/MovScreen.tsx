import React from 'react';

import ZoneLocation from '../containers/ZoneLocation';
import DisplayScreen from './DisplayScreen';

const MovScreen = () => (
  <DisplayScreen>
    <ZoneLocation />
  </DisplayScreen>
);

export default MovScreen;
