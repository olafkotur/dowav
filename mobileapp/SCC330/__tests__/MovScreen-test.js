import 'react-native';
import React from 'react';

import renderer from 'react-test-renderer';
import MovScreen from '../screens/MovScreen';

it('renders temperature screen correctly', () => {
  const component = renderer.create(<MovScreen />).toJSON();
  expect(component).toMatchSnapshot();
});
