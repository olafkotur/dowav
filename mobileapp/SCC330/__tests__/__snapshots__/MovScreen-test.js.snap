// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders temperature screen correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#2c5364",
      "flex": 1,
      "justifyContent": "center",
      "padding": "5%",
    }
  }
>
  <View
    onLayout={[Function]}
    style={
      Object {
        "flex": 1,
      }
    }
  />
</View>
`;
